flow/designs/gf12/ariane/metadata-hier_rtlmp-ok.json

```diff
@@ -14,106 +14,106 @@
+  "cts__timing__setup__tns__pre_repair": -489562.53,
   "cts__timing__drv__max_slew__pre_repair": 30.0,
   "cts__timing__drv__max_fanout__pre_repair": 0.0,
   "cts__timing__drv__max_cap__pre_repair": 0.0,
-  "cts__timing__setup__ws__pre_repair": -1403.18,
-  "cts__timing__setup__tns__post_repair": -3893940.75,
-  "cts__timing__setup__ws__post_repair": -1403.13,
+  "cts__timing__setup__ws__pre_repair": -403.18,
+  "cts__timing__setup__tns__post_repair": -489468.0,
+  "cts__timing__setup__ws__post_repair": -403.13,
   "cts__timing__drv__max_slew__post_repair": 30.0,
   "cts__timing__drv__max_fanout___post_repair": 0.0,
   "cts__timing__drv__max_cap__post_repair": 0.0,
-  "cts__timing__setup__tns": -2175367.5,
-  "cts__timing__setup__ws": -854.79,
+  "cts__timing__setup__tns": 0.0,
+  "cts__timing__setup__ws": 4.03,
   "cts__timing__drv__max_slew": 36.0,
   "cts__timing__drv__max_fanout": 0.0,
-  "cts__design__instance__count__hold_buffer": 4309.0,
-  "globalroute__clock__latency__min": 229.43,
-  "globalroute__clock__latency__max": 1689.89,
-  "globalroute__clock__skew__worst": 1448.5,
-  "globalroute__timing__setup__tns": -1430057.12,
-  "globalroute__timing__setup__ws": -620.71,
-  "globalroute__timing__drv__max_slew": 156.0,
+  "cts__design__instance__count__hold_buffer": 4166.0,
+  "globalroute__clock__latency__min": 228.35,
+  "globalroute__clock__latency__max": 1702.3,
+  "globalroute__clock__skew__worst": 1462.21,
+  "globalroute__timing__setup__tns": 0.0,
+  "globalroute__timing__setup__ws": 259.95,
+  "globalroute__timing__drv__max_slew": 146.0,
   "globalroute__timing__drv__max_fanout": 0.0,
   "globalroute__timing__drv__max_cap": 0.0,
-  "globalroute__timing__clock__slack": -620.712,
-  "globalroute__timing__clock__period": 2489.677,
-  "globalroute__runtime__total": "2:46.74",
-  "globalroute__cpu__total": 162.93,
-  "globalroute__mem__peak": 2707904.0,
-  "detailedroute__route__wirelength": 3014202.0,
-  "detailedroute__route__vias": 2038678.0,
+  "globalroute__timing__clock__slack": 259.946,
+  "globalroute__timing__clock__period": 2603.051,
+  "globalroute__runtime__total": "4:01.67",
+  "globalroute__cpu__total": 237.55,
+  "globalroute__mem__peak": 2706904.0,
+  "detailedroute__route__wirelength": 3029673.0,
+  "detailedroute__route__vias": 2104466.0,
   "detailedroute__route__drc_errors": 0,
-  "detailedroute__runtime__total": "1:31:30",
-  "detailedroute__cpu__total": 32046.25,
-  "detailedroute__mem__peak": 24731520.0,
-  "finish__clock__latency__min": 207.92,
-  "finish__clock__latency__max": 1179.37,
-  "finish__clock__skew__worst": 957.71,
+  "detailedroute__runtime__total": "1:11:41",
+  "detailedroute__cpu__total": 27498.11,
+  "detailedroute__mem__peak": 24370956.0,
+  "finish__clock__latency__min": 206.74,
+  "finish__clock__latency__max": 1166.6,
+  "finish__clock__skew__worst": 946.27,
   "finish__timing__setup__tns": 0.0,
-  "finish__timing__setup__ws": 150.68,
+  "finish__timing__setup__ws": 1103.38,
```

Comment:
should we tighten more?

Reply:
Hi Vitor -- We will tighten all of these designs in the next revision.  Also when ratcheting the SDC I try to make the global route timing close to 0.